src/test/java/trackitnus/logic/parser/contact/AddContactCommandParserTest.java

```diff
@@ -90,17 +90,17 @@ public void parse_compulsoryFieldMissing_failure() {
                 + ContactCommandTestUtil.EMAIL_DESC_BOB + ContactCommandTestUtil.ADDRESS_DESC_BOB,
             expectedMessage);
 
-        // missing phone prefix
-        assertParseFailure(parser,
-            ContactCommandTestUtil.NAME_DESC_BOB + ContactCommandTestUtil.VALID_PHONE_BOB
-                + ContactCommandTestUtil.EMAIL_DESC_BOB + ContactCommandTestUtil.ADDRESS_DESC_BOB,
-            expectedMessage);
-
-        // missing email prefix
-        assertParseFailure(parser,
-            ContactCommandTestUtil.NAME_DESC_BOB + ContactCommandTestUtil.PHONE_DESC_BOB
-                + ContactCommandTestUtil.VALID_EMAIL_BOB + ContactCommandTestUtil.ADDRESS_DESC_BOB,
-            expectedMessage);
+//        // missing phone prefix
+//        assertParseFailure(parser,
+//            ContactCommandTestUtil.NAME_DESC_BOB + ContactCommandTestUtil.VALID_PHONE_BOB
+//                + ContactCommandTestUtil.EMAIL_DESC_BOB + ContactCommandTestUtil.ADDRESS_DESC_BOB,
+//            expectedMessage);
+
+//        // missing email prefix
+//        assertParseFailure(parser,
+//            ContactCommandTestUtil.NAME_DESC_BOB + ContactCommandTestUtil.PHONE_DESC_BOB
+//                + ContactCommandTestUtil.VALID_EMAIL_BOB + ContactCommandTestUtil.ADDRESS_DESC_BOB,
+//            expectedMessage);
```

Comment:
Why are these lines commented out?

Reply:
because now missing phone is not an error any more, later on I will convert them to positive testcases